### PR TITLE
Corrected Platform version

### PR DESCRIPTION
**Pull request by ianf77**
Changed AAP version from 2.2 to 2.3

Upgrade Docs for AAP 2.3 not fully updated

https://issues.redhat.com/browse/AAP-7332

### CODE REVIEW

File: downstream/modules/platform/proc-choosing-obtaining-installer.adoc [hunk 1]
```diff
@@ -34,13 +34,13 @@ $ tar xvzf ansible-automation-platform-setup-<latest-version>.tar.gz
 v.2.2 for RHEL 8 for x86_64
```
Comment (emurtoug) [inline downstream/modules/platform/proc-choosing-obtaining-installer.adoc:34]: Should this be changed to 'v.2.3 for RHEL 8'?

File: downstream/modules/platform/proc-choosing-obtaining-installer.adoc [hunk 2]
```diff
@@ -34,13 +34,13 @@ $ tar xvzf ansible-automation-platform-setup-<latest-version>.tar.gz
...
 +
 ----
-$ sudo dnf install --enablerepo=ansible-automation-platform-2.2-for-rhel-8-x86_64-rpms ansible-automation-platform-installer
+$ sudo dnf install --enablerepo=ansible-automation-platform-2.3-for-rhel-8-x86_64-rpms ansible-automation-platform-installer
 ----
 +
 v.2.2 for RHEL 9 for x86-64
```
Comment (emurtoug) [inline downstream/modules/platform/proc-choosing-obtaining-installer.adoc:40]: Same comment as above.

File: downstream/modules/platform/proc-choosing-obtaining-installer.adoc (hunk 1 above)
Comment (ianf77) [reply downstream/modules/platform/proc-choosing-obtaining-installer.adoc:34]: Not sure about that. I wonder if there's a corresponding version for RHEL versions. I'll check.

**Review by emurtoug — APPROVED** [review]
LGTM

**Pull request merged**